src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -193,11 +193,11 @@ public static partial class JsonSerializer
         public static object? Deserialize(System.ReadOnlySpan<byte> utf8Json, System.Type returnType, System.Text.Json.JsonSerializerOptions? options = null) { throw null; }
         public static object? Deserialize(System.ReadOnlySpan<byte> utf8Json, System.Type returnType, System.Text.Json.Serialization.JsonSerializerContext context) { throw null; }
```

Comment:
If/when C# supports utf8 literals, we'll presumably want to attribute these utf8json parameters as well.